model/Core/Vocabularies/HashAlgorithm.md

```diff
@@ -37,4 +40,5 @@ and is a one-way function, that is, a function which is practically infeasible t
 - sha512: TODOdescription
 - spdxPvcSha1: TODOdescription
 - spdxPvcSha256: TODOdescription
+- sphincs+: TODOdescription
```

Comment:
`sphincsPlus`
and probably also
`sphincsPlusShake256`
`sphincsPlusSha256`
`sphincsPlusHaraka`